### DISCUSSION TITLE
Login Fail: " An error has occurred. Could not send login notification email. Please contact your administrator."

### DISCUSSION THREAD

**Discussion by ph0** [dani-garcia/vaultwarden#2457] [Q&A]
I am fully updated.

This happened before.

I click login and it tells me a login notification email failed without letting me login.

I can login to admin with the admin token from my config.json. I force resync of clients and that did nothing.

This is the second time this has happened. Last time 14 hours passed and it suddenly started working again with no explanation as to why. I have rebooted my server. Same result.

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.24.0
* Web-vault version: v2.25.1
* Running within Docker: true (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: false
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.35.4
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** DOMAIN, SIGNUPS_ALLOWED, ADMIN_TOKEN

```json
{
  "_duo_akey": null,
  "_enable_duo": false,
  "_enable_email_2fa": true,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_ip_header_enabled": true,
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_max_conns": 10,
  "database_url": "****/**.*******",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://*********.******.***.******",
  "domain_origin": "*****://*********.******.***.******",
  "domain_path": "",
  "domain_set": true,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_attempts_limit": 3,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 5 * * * *",
  "emergency_request_timeout_schedule": "0 5 * * * *",
  "enable_db_wal": true,
  "extended_logging": true,
  "helo_name": "****.***.******",
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_org_name": "Vaultwarden",
  "invitations_allowed": true,
  "ip_header": "X-Real-IP",
  "job_poll_interval_ms": 30000,
  "log_file": null,
  "log_level": "Info",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "password_iterations": 100000,
  "reload_templates": false,
  "require_device_email": true,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": true,
  "signups_allowed": false,
  "signups_domains_whitelist": "",
  "signups_verify": false,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": null,
  "smtp_debug": false,
  "smtp_explicit_tls": false,
  "smtp_from": "******@***.******",
  "smtp_from_name": "Vaultwarden",
  "smtp_host": "****.***.******",
  "smtp_password": "***",
  "smtp_port": 587,
  "smtp_ssl": true,
  "smtp_timeout": 15,
  "smtp_username": "******@***.******",
  "templates_folder": "data/templates",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_syslog": false,
  "user_attachment_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": true,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```
</details>

**Comment by BlackDex** [comment]
You have `require_device_email` enabled which prevents a login if sending a new device email failed. I would check your smtp settings or smtp server.

14 hours seems like a strange number, but could be some mail server timeout setting or whatever.

**Comment by ph0** [comment]
Can you explain further? I have used the same email/pass from the same device for more than 1 year now with no issues. Also, I can login and check nothing because my passwords are stored in Vaultwarden.

**Comment by ph0** [comment]
I sent a test email from admin SMTP settings and got this error: "Error sending SMTP test email
SMTP Connection error: error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../ssl/record/ssl3_record.c:332:"

**Comment by ph0** [comment]
I must note my login was working until the moment I cleared my browser history and cache.

**Reply by BlackDex** [reply]
Yes, that i understand, since with your browser cache not cleared, Vaultwarden doesn't see the login as a new device, and no mail is being send, and thus no issue.

**Comment by BlackDex** [comment]
Well either something is not configured correctly for sending emails.
Or the mail server you are using has issues, like a bad certificate, or something. But that is just guessing.

The only thing i know for sure is that Vaultwarden can't send a mail, and that is causing you to not be able to login.

Disabled `require_device_email` and try again.
This at least isn't a Vaultwarden issues it self.

**Comment by ph0** [comment]
I disabled require_device_email and got the same result.

**Reply by BlackDex** [reply]
Not sure how you disabled it. Via the admin interface or docker envs?
Just be sure to check the admin interface if it is actually set

**Comment by ph0** [comment]
I disabled it in the config.json and that did not work.

My IT guy just disabled it in the docker and login now works because it overwrites the one on the cpanel

What I want to know is why did this take a year to stop me from logging in?

**Reply by BlackDex** [reply]
Because something is wrong with the mail server you are using. Either wrong configuration or something else. That is not something we know. Ask your mail server administrator if they can check the logs.

**Comment by ph0** [comment]
"Basically the cookie that had all the information to not make a “new device”" is what triggered the login issue

**Reply by BlackDex** [reply]
Well it triggered the mail sending, which triggered the issue.
